functions/src/api/middlewares/authenticateJWTMiddleware.spec.ts

```diff
@@ -0,0 +1,127 @@
+// tslint:disable-next-line
+import '../../testUtils/mockFirebaseAdminAndFunctions'
+import { firestoreGet } from '../../testUtils/firestoreStub'
+
+import { authenticateJWTMiddleware } from './authenticateJWTMiddleware'
+import {
+    ForbiddenError,
+    PreconditionFailedError,
+    UnauthorizedError,
+} from '../errors/HttpError'
+
+describe('authentificateJWTMiddleware', () => {
+    const mockNext = jest.fn()
+    const mockRes = jest.fn()
+
+    afterEach(() => {
+        mockNext.mockClear()
+        mockRes.mockClear()
+    })
+
+    it('should not be authorized due to missing authorization header', () => {
```

Comment:
I suggest to use consistent test descriptions. I would go with the affirmative sentence like below:

```
should fail due to missing authorization header
```